### PR TITLE
Update crew and install.sh to remove the need for sudo

### PR DESCRIPTION
**Pull request by uberhacker**
This is a reissue of PR #176. Installs cleanly on a power washed Chromebook with specs below:

Version 56.0.2924.110 (64-bit)
Platform 9000.91.0 (Official Build) stable-channel kip
Firmware Google_Kip.5216.227.78

### CODE REVIEW

**Comment by uberhacker** [comment]
@thedamian: I reopened this PR since I had trouble updating the previous #546.  I updated the crew version, so we will also need to tag a new [release](https://github.com/skycocker/chromebrew/releases).  We probably need a new release anyway since there have been so many PRs merged into master recently. :)

File: install.sh [hunk 1]
```diff
@@ -96,7 +99,7 @@ function extract_install () {
     rm -rf ./usr
     tar -xf $2
     echo "Installing $1 (this may take a while)..."
-    sudo tar cf - ./usr/* | (cd /; sudo tar xp --keep-directory-symlink -f -)
+    tar cf - ./usr/*
```
Comment (uberhacker) [inline install.sh]: Do we need this?  From what I can tell, it serves no purpose and will never execute anyway.

**Comment by uberhacker** [comment]
@jam7:  It took me awhile to understand what was going on here but now I get it.  I thought initially we could just simply extract the files but the original logic was trying to preserve the symlinks.  I have pushed the change you suggested.  Please test again when you get a chance.
Thanks

**Comment by jam7** [comment]
It looks like working great here on armv7l.  #657 was needed to apply, though.  Nice work.

**Comment by uberhacker** [comment]
@lyxell: We'll need to tag a new [release](https://github.com/skycocker/chromebrew/releases) since the crew version changed, right?

**Pull request merged**